tests/scripts/run-throughput-perf.py

```diff
@@ -0,0 +1,361 @@
+#!/usr/bin/env python
+#
+# Licensed to the .NET Foundation under one or more agreements.
+# The .NET Foundation licenses this file to you under the MIT license.
+# See the LICENSE file in the project root for more information.
+#
+##########################################################################
+##########################################################################
+#
+# Module: run-corefx-tests.py
+#
+# Notes:
+#
+#
+##########################################################################
+##########################################################################
+
+import argparse
+import distutils.dir_util
+import os
+import re
+import shutil
+import subprocess
+import sys
+import time
+
+##########################################################################
+# Globals
+##########################################################################
+
+# List of dlls we want to crossgen
+dll_list = [
+        "System.Private.CoreLib",
+        "System.Reflection.Metadata",
+        "System.Linq.Expressions",
+        "Microsoft.CSharp",
+        "System",
+        "Microsoft.CodeAnalysis.VisualBasic",
+        "System.Private.DataContractSerialization",
+        "System.Core",
+        "System.Xml",
+        "Microsoft.CodeAnalysis.CSharp",
+        "Microsoft.CodeAnalysis",
+        "System.Linq.Parallel",
+        "System.Private.Xml"
```

Comment:
Though perhaps I should add a comment as to why these assemblies were chosen, which is that they take at least 1 second or longer to crossgen.